src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -670,14 +694,15 @@ func (c *Client) GetBigBook(ctx context.Context, req *librarypb.GetBookRequest)
 }
 
 // GetBigNothing test long-running operations with empty return type.
-func (c *Client) GetBigNothing(ctx context.Context, req *librarypb.GetBookRequest) (*GetBigNothingOperation, error) {
+func (c *Client) GetBigNothing(ctx context.Context, req *librarypb.GetBookRequest, opts ...gax.CallOption) (*GetBigNothingOperation, error) {
     ctx = insertXGoog(ctx, c.xGoogHeader)
+    opts = append(c.CallOptions.GetBigNothing[0:len(c.CallOptions.GetBigNothing):len(c.CallOptions.GetBigNothing)], opts...)
     var resp *longrunningpb.Operation
     err := gax.Invoke(ctx, func(ctx context.Context, settings gax.CallSettings) error {
         var err error
         resp, err = c.client.GetBigNothing(ctx, req, settings.GRPC...)
         return err
-    }, c.CallOptions.GetBigNothing...)
+    }, opts...)
```

Comment:
We resolve the call option for getting the first operation. We don't have call options for calls to Poll/Wait though, since the longrunning package itself doesn't use gapic-generated code. Are you OK with this?